app/src/main/java/com/greenfox/tribesoflagopusandroid/LoginActivity.java

```diff
@@ -41,7 +44,7 @@ public void login(View view) {
         Intent intent = new Intent(this, MainActivity.class);
         startActivity(intent);
```

Comment:
@georgezsiga you should only store the data and start the `MainActivity` if the login was successful